### PR TITLE
Feature/section table

### PR DESCRIPTION
**Pull request by NicholasLYang**

### CODE REVIEW

File: src/js/modules/articles/selectors.js [hunk 1]
```diff
@@ -4,12 +4,12 @@
 import { createSelector } from "reselect";
 import { usersSelector } from "../users/selectors";
 
-export const articlesSelector = state => state.articles.list;
+export const getSelectedArticles = state => state.articles.list;
```
Comment (squirrelsquirrel78) [inline src/js/modules/articles/selectors.js]: Inaccurate name.

File: src/js/modules/sections/actionTypes.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
-export const FETCH_SECTIONS_REQUESTED = "forms/FETCH_SECTIONS_REQUESTED";
-export const FETCH_SECTIONS_SUCCEEDED = 'forms/FETCH_SECTIONS_SUCCEEDED';
-export const FETCH_SECTIONS_FAILED = 'forms/FETCH_SECTIONS_FAILED';
+export const FETCH_SECTIONS_REQUESTED = "sections/FETCH_SECTIONS_REQUESTED";
+export const FETCH_SECTIONS_SUCCEEDED = 'sections/FETCH_SECTIONS_SUCCEEDED';
+export const FETCH_SECTIONS_FAILED = 'sections/FETCH_SECTIONS_FAILED';
```
Comment (squirrelsquirrel78) [inline src/js/modules/sections/actionTypes.js]: This is small but...doubles + singles quotations lol

File: src/js/modules/sections/actions.js [hunk 3]
```diff
@@ -36,20 +31,51 @@ export const createSection = section => dispatch => {
...
-      type: t.CREATE_SECTION_FAILED,
-      payload: error
+    .post(`${STUY_SPEC_API_URL}/sections`, section, AXIOS_CONFIG)
+    .then(response => {
+      dispatch({
+        type: t.CREATE_SECTION_SUCCEEDED,
+        payload: response.data
+      });
+      dispatch(fetchSections());
+      dispatch(push("/"));
```
Comment (squirrelsquirrel78) [inline src/js/modules/sections/actions.js]: As a user, I want to be redirected to the SectionTable after I finish creating a section.

**Review by squirrelsquirrel78 — CHANGES_REQUESTED** [review]

File: src/js/modules/articles/selectors.js (hunk 1 above)
Comment (NicholasLYang) [reply src/js/modules/articles/selectors.js]: Oops idk why that happened

**Pull request merged**